packages/module/patternfly-docs/content/examples/catalogTile.css

```diff
@@ -21,24 +17,21 @@
   color: var(--pf-t--global--text--color--link--hover);
   text-decoration: underline;
 }
-.ws-react-e-catalog-tile .pf-v6-c-card__actions {
-  padding-left: 5px;
-}
 .ws-react-e-catalog-tile .catalog-tile-pf-icon {
-  font-size: 40px;
-  height: 40px;
+  font-size: 37px;
+  height: 37px;
   max-width: 60px;
-  min-width: 40px;
+  min-width: 37px;
 }
 .ws-react-e-catalog-tile .catalog-tile-pf-badge-container {
   display: flex;
   flex: 1;
   justify-content: flex-end;
-  margin-left: 10px;
+  margin-inline-start: var(--pf-t--global--spacer--sm);
 }
 .ws-react-e-catalog-tile .catalog-tile-pf-badge-container .catalog-tile-pf-badge {
-  font-size: 16px;
-  margin-left: 5px;
+  font-size: var(--pf-t--global--font--size--body--default);
+  margin-inline-start: var(--pf-t--global--spacer--xs);
 }
```

Comment:
I'm not sure what all goes in this element but depending on what all it supports, this may be able to be improved.